runtime/src/runtime/fvm.rs

```diff
@@ -453,10 +458,16 @@ where
         fvm::actor::install_actor(code_id).map_err(|_| Error::msg("failed to install actor"))
     }
 
-    fn hash(&self, hasher: fvm_shared::crypto::hash::SupportedHashes, data: &[u8]) -> Vec<u8> {
+    fn hash(&self, hasher: SupportedHashes, data: &[u8]) -> Vec<u8> {
         fvm::crypto::hash_owned(hasher, data)
     }
 
+    fn hash_64(&self, hasher: SupportedHashes, data: &[u8]) -> ([u8; 64], usize) {
```

Comment:
```suggestion
    fn hash_fixed<const S: usize>(&self, hasher: SupportedHashes, data: &[u8]) -> ([u8; S], usize) {
```

(if we're going to introduce this method)

Reply:
This is what I tried originally, but multisig actor expects to use `&dyn Primitives`, adding a generic parameter breaks object safety.

We could change that though.